testpmd-lb-operator/roles/loadbalancer/tasks/main.yml

```diff
@@ -24,6 +24,10 @@
   set_fact:
     networks: "{{ packet_generator_networks + cnf_app_networks }}"
 
+- name: print networks
+  debug:
```

Comment:
Should we keep the debug? I don't mind but it might be a sensible information.

Reply:
I think it's not harmful, this info can be also extracted from must-gather probably, by checking the resources created on the cluster, and we're also submitting the must-gather to the DCI job, so it would be the same kind of information

Reply:
I like the debug, I think we can go with it for now, and maybe in the future we could add a boolean to enable/disable those debug tasks.